config/genesis.go

```diff
@@ -0,0 +1,145 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+
+	apiConfig "github.com/spacemeshos/go-spacemesh/api/config"
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/hash"
+)
+
+const (
+	ExtraDataLen                 = 255
+	DefaultGenesisConfigFileDir  = "data"
+	DefaultGenesisConfigFileName = "genesis.conf"
+)
+
+type GenesisConfig struct {
+	Accounts    *apiConfig.GenesisAccountConfig `mapstructure:"genesis-api"`
+	GenesisTime string                          `mapstructure:"genesis-time"`
+	ExtraData   string                          `mapstructure:"genesis-extradata"`
+	GoldenATXID string
+	GenesisID   string
+	Path        string
```

Comment:
this config should have only 3 things: Accounts (mapping), GenesisTime, ExtraData

remove everything else

Reply:
done